packages/graphback-datasync/src/util.ts

```diff
@@ -39,3 +44,35 @@ export const DataSyncFieldNames = {
   lastUpdatedAt: '_lastUpdatedAt',
   deleted: '_deleted'
 }
+
+
+
+export interface DataSyncModelConflictConfig {
+  enabled: boolean
+  conflictResolution?: ConflictResolutionStrategy
+}
+
+export interface DataSyncModelConfigMap {
+  [modelName: string]: DataSyncModelConflictConfig
+}
+
+export interface ConflictResolutionStrategy {
+  resolveUpdate(conflict: ConflictMetadata): any
+}
+
+export const ServerSideConflictResolution: ConflictResolutionStrategy = {
```

Comment:
I would recomend to provide names for strategies. ServerFirst, LastUpdatedFirst etc.